server/chat/backend/agent/utils/model_cutoff_manager.py

```
@@ -64,11 +64,32 @@ def cutoff_date(year: int, month: int, day: int = 1) -> datetime:
             ),
         }
 
-        # Google Models (via OpenRouter)
+        # Google / Vertex AI Models
         google_models = {
             "google/gemini-3.1-pro-preview": ModelInfo(
                 "gemini-3.1-pro-preview", "google", cutoff_date(2025, 4, 1), True, True
```

Ruff (0.15.17)

[warning] Boolean positional value in function call

(FBT003)